app/src/main/java/dev/leonlatsch/photok/gallery/ui/GalleryFragment.kt

```diff
@@ -94,4 +158,26 @@ class GalleryFragment : Fragment() {
         viewModel.checkForNewFeatures()
 
     }
+
+
+    @Composable
+    private fun ImportDialogue(content: String = "", onPositiveTextOnClick: ()-> Unit, onNegativeTextOnClick: ()-> Unit) {
```

Comment:
There already is a compose implementation for confirmation dialogs.

Please use it instead